recipe/meta.yaml

```diff
@@ -7,22 +7,50 @@ package:
 source:
   fn: libsodium-{{ version }}.tar.gz
   url: https://github.com/jedisct1/libsodium/releases/download/{{ version }}/libsodium-{{ version }}.tar.gz
-  sha1: f34f78330cf1a4f69acce5f3fc2ada2d4098c7f4
+  sha256: fb6a9e879a2f674592e4328c5d9f79f082405ee4bb05cb6e679b90afe9e178f4
 
 build:
-  skip: true    # [win]
+  skip: true  # [win and py>35]
   number: 0
+  features:
+    - vc9     # [win and py27]
+    - vc10    # [win and py34]
+    - vc14    # [win and py>=35]
+
+requirements:
+  build:
+    - python      # [win]
+    - msinttypes  # [win and py27]
+  run:
+    - vc 9   # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py>=35]
```

Comment:
libsodium is only a C library. It shouldn't be running with a matrix of Python versions. It should only use one of these.

Reply:
@minrk  pynacl uses libsodium and since we tie various python versions with various vc versions, I think we need to do this?

https://github.com/pyca/pynacl/issues/165

Reply:
Not sure, but this would probably also help:
 https://lists.zeromq.org/pipermail/zeromq-dev/2014-October/027075.html

Reply:
So the matrix here is just to ensure the right VC runtime is used on Windows. This is important as different Python versions uses different VC runtimes. If a user ends up using two pieces of software built with different VC runtimes that share memory, file handles, or some other things, they will crash. So this is just trying to ensure the VC runtimes are properly aligned so that different linked software don't run into issues. That said we can instantiate this matrix with other mechanisms. Just turns out to be easiest with `python`.

Reply:
Thanks for the explanation.